src/fiatconnect/saga.ts

```diff
@@ -445,14 +459,54 @@ export function* _getSpecificQuote({
     providerIds: [providerId],
   })
   const normalizedQuotes = normalizeFiatConnectQuotes(flow, quotes)
-  const normalizedQuote = normalizedQuotes.find((q) => q.getFiatAccountType() === fiatAccountType)
+
+  // If no account was provided, we need to check cached accounts for a possible match, and fetch the obfuscated account
+  // from the selected provider.
+  let selectedFiatAccount = fiatAccount
+  if (!selectedFiatAccount) {
+    const cachedFiatAccounts: CachedFiatAccountUse[] = yield select(cachedFiatAccountUsesSelector)
+    // If defined, cachedFiatAccount is a Fiat Account whose providerId and fiatAccountType match one of our quotes.
```

Comment:
eventually, we may allow users to link multiple accounts for the same provider and fiat account type. could be good to leave a note here indicating explicitly that that isn't supported right now. Or you could try supporting it, but that seems like a hassle.

Reply:
seems like you'd need to cache both the quote and the fiat account ID the user was trying to create a transfer with.

Reply:
Yes; we cache the quote as soon as the user has submitted KYC data to IHL, before the user enters fiat account information, so it would make things a bit tricky. We could probably do something like cache the quote when KYC is posted, and update it with fiat account info if the user submits a fiat account while still in the same CICO flow... I'll leave a note saying multiple accounts are not yet supported here.